### PR TITLE
Minor correction in output format

### PR DESCRIPTION
**Pull request by kaganisildak**
Comma should not come after the plugin value in KV format(Other plugins follow this rule)

### CODE REVIEW

**Comment by tklengyel** [comment]
@drakvuf-jenkins This is OK to test

**Comment by kaganisildak** [comment]
@drakvuf-jenkins Retest this please

File: src/plugins/memdump/userhook.cpp [hunk 1]
```diff
@@ -171,16 +171,16 @@ static event_response_t usermode_return_hook_cb(drakvuf_t drakvuf, drakvuf_trap_
...
+            printf("memdump-userhook," FORMAT_TIMEVAL ",%" PRIu32 ",0x%" PRIx64 ",\"%s\",%" PRIi64 ",\"%s\",0x%" PRIx64 ",\"",
                    UNPACK_TIMEVAL(info->timestamp), info->vcpu, info->regs->cr3, info->proc_data.name,
                    info->proc_data.userid, info->trap->name, info->regs->rax, info->regs->rip);
             print_arguments(ret_target->arguments);
             printf("\"");
             break;
         case OUTPUT_KV:
-            printf("memdump, Time=" FORMAT_TIMEVAL ",VCPU=%" PRIu32 ",CR3=0x%" PRIx64 ",ProcessName=\"%s\",UserID=%" PRIi64 ",Method=\"%s\",CalledFrom=0x%" PRIx64 ",ReturnValue=0x%" PRIx64 ",Arguments=\"",
-                   UNPACK_TIMEVAL(info->timestamp), info->vcpu, info->regs->cr3, info->proc_data.name,
-                   info->proc_data.userid, info->trap->name, info->regs->rip, info->regs->rax);
```
Comment (icedevml) [inline src/plugins/memdump/userhook.cpp]: This change drops the feature of printing API call return value.

Comment (kaganisildak) [reply src/plugins/memdump/userhook.cpp]: I had to open a new pr because i cant modify code in repo I deleted.
#815

**Pull request closed without merging**